types/src/error.rs

```diff
@@ -38,6 +38,18 @@ pub enum CallError {
 	},
 }
 
+// TODO(niklasad1): doesn't work probably conflicting with `thiserror::Error`
```

Comment:
I was wrong, essentially you can't have this if the type implements `std::error::Error` :cold_face:

Reply:
hrm, go with `From<anyhow::Error>` here?